src/pages/questions/ui/QuestionsPage.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { useState, useEffect } from 'react';
+import { NavLink, Navigate } from 'react-router-dom';
+import cl from './QuestionsPage.module.scss';
+import PhoneImg from 'shared/assets/img/phone.svg';
+import { Mark } from 'widgets/mark';
+import { Button } from 'shared/ui/button/Button';
+import { useAppSelector } from 'shared/lib/hooks/useAppSelector/useAppSelector';
+import { useAppDispatch } from 'shared/lib/hooks/useAppDispatch/useAppDispatch';
+import { fetchQuestions } from 'entities/questions';
+import { getCompletedSurvey, getMain, getResult } from 'app/providers/router/routeConfig/routes';
+import { useQuestionsCompleted } from 'shared/lib/hooks/useQuestionsCompited/useQuestionsCompited';
+
+const QuestionsPage: React.FC = () => {
+    const { list, error } = useAppSelector((state) => state.questions);
+    const answersList = useAppSelector((state) => state.answers.list);
+    
+    const [btnState, setBtnState] = useState(true);
+    
+    const dispatch = useAppDispatch();
+    useEffect(() => {
+        dispatch(fetchQuestions());
+    }, [dispatch]);
+    
+    useEffect(() => {
+        answersList.length === 6 ?  setBtnState(false) : setBtnState(true)
```

Comment:
```suggestion
        answersList.length === list.length ?  setBtnState(false) : setBtnState(true)
```
тут лучше не хардкодить значение, а взять длину списка вопросов